app/assets/stylesheets/modules/item-selector.scss

```diff
@@ -86,3 +86,34 @@
   }
 }
 
+.bound-withs {
+  max-height: 200px;
+  background-color: #F4F4F4;
+  overflow: scroll;
+  padding: 8px;
+  margin-bottom: 15px;
+  border-radius: 4px;
+
+  .bound-with-item span{
+    display: block;
+  }
+
+  .bound-with-item {
+    padding: 4px 0px;
+  }
+
+  .heading {
+    font-weight: 500;
+    margin-bottom: 10px;
+  }
+
+  li:not(:only-child) {
+    border-top: 1px solid #D5D5D4;
```

Comment:
Please make this color a variable and put it in https://github.com/sul-dlss/sul-requests/blob/main/app/assets/stylesheets/sul-variables.scss#L1